src/Compilers/Core/Portable/PEWriter/Members.cs

```diff
@@ -70,6 +70,23 @@ internal enum CallingConvention
         ExplicitThis = SignatureAttributes.ExplicitThis,
     }
 
+    internal static class CallingConventionUtils
+    {
+        private const SignatureCallingConvention SignatureMask =
+            SignatureCallingConvention.Default
+            | SignatureCallingConvention.CDecl
+            | SignatureCallingConvention.StdCall
+            | SignatureCallingConvention.ThisCall
+            | SignatureCallingConvention.FastCall
+            | SignatureCallingConvention.VarArgs;
+
+        internal static CallingConvention FromSignatureConvention(this SignatureCallingConvention convention)
+            => (CallingConvention)(convention & SignatureMask);
```

Comment:
>convention & SignatureMask [](start = 35, length = 26)

Are we silently ignoring bits outside `SignatureMask`? Should we `throw UnsupportedSignatureContent()` in those cases instead?